pommerman/utility.py

```diff
@@ -326,3 +328,34 @@ def get_next_position(position, direction):
 
 def make_np_float(feature):
     return np.array(feature).astype(np.float32)
+
+def join_json_state(record_json_dir, agents, finishedAt, config):
+    josnSchema = {
```

Comment:
name

Reply:
typical. 😹It wouldn't be a proper PR if I didn't misspell something.